launchUI.py

```diff
@@ -42,8 +39,8 @@ def window_size(window, width, height):
     window.geometry(f"{width}x{height}")
 master = tkinter.Tk()
 master.title("CGRA-Flow: An Integrated End-to-End Framework for CGRA Exploration, Compilation, and Development")
-default_width = 1725
-default_height = 970
+default_width = 1723
+default_height = 900

```

Comment:
Is this `default_width` and `default_height` for the entire window? If so, can you add comment above it to mention it is for the entire window?

Also, can you please try to decrease the upper part height? For now, the upper part (e.g., CGRA panel) has much redundant space.